### ISSUE TITLE
User input during playback is also used, and messes up the playback

### ISSUE THREAD

**Issue by alice-i-cecile** [Leafwing-Studios/leafwing_input_playback#13]
## What you did

Ran the `input_playback` example.

## What you expected to happen

No user input would be accepted during playback, and I get a perfect recording.

## What actually happened

The input that I entered was also applied, messing up the results.

**Comment by bzm3r** [comment]
Note: this is different from how one might disable user input during a cutscene. In situations like that, you can simply disable the system that responds to user input, effectively ignoring it. 

Here, we want to ignore user input at a much more basic level.

**Comment by snendev** [comment]
has anything on the bevy side changed here since 2022?

from a short bit of code delving in the bevy repo, it seems like either:

1) bevy_input needs to provide some control for differentiating the "sources" of inputs (winit vs something else, would need to be provided at event construction which makes the types more complicated)
or
2) bevy_winit needs to provide an off switch for forwarding certain events

neither of these feels good to me right now.

(just posting here since I'm thinking on it -- I'll raise a question in the discord if I don't get anywhere.)

**Comment by alice-i-cecile** [comment]
Yeah, not much has changed here. I think you could add some carefully timed event draining systems and get this to work without any changes upstream though.

**Comment by snendev** [comment]
> I think you could add some carefully timed event draining systems

Great point. `playback_timestamped_input` can occur in `PreUpdate` after `InputSystems` and a drain system, and it shouldn't really change much else. Thanks!
